index.yaml

```diff
@@ -375,7 +473,7 @@ entries:
       catalog.cattle.io/ui-component: longhorn
     apiVersion: v1
     appVersion: v1.1.2
-    created: "2021-09-29T14:12:43.175318108-04:00"
+    created: "2021-08-25T15:50:58.486087084-07:00"
```

Comment:
Note: Longhorn 1.1 version is correct, it is not actually being released. We just moved longhorn to its own directory but made no other changes. Therefore, a new asset under `assets/longhorn-1.1` for `100.0.0+up1.1.2` is expected (which matches this: https://github.com/rancher/charts/tree/release-v2.6/charts/longhorn/longhorn/100.0.0%2Bup1.1.2), not `100.0.1+up1.1.2`. 

The index.yaml should also not be modified for this additional chart.

Reply:
@cbron should we consider putting `doNotTrack` on the longhorn 1.1 chart in order to avoid releasing a second, identical version of the same chart? It should have no impact since our index.yaml does not reference it, but we could avoid generating the new chart just for good measure.

Reply:
Done